src/hotspot/cpu/s390/vm_version_s390.hpp

```diff
@@ -513,7 +513,6 @@ class VM_Version: public Abstract_VM_Version {
   static void set_has_CompareBranch()             { _features[0] |= GnrlInstrExtFacilityMask; }
   static void set_has_CompareTrap()               { _features[0] |= GnrlInstrExtFacilityMask; }
   static void set_has_RelativeLoadStore()         { _features[0] |= GnrlInstrExtFacilityMask; }
-  static void set_has_GnrlInstrExtensions()       { _features[0] |= GnrlInstrExtFacilityMask; }
```

Comment:
I know this PR is still in draft state. Just a thought: I would like to keep the methods in `vm_version_s390.hpp` file for now. I'm planning to remove the checks applicable to older hardware. So it would be better, If I clean these methods as a part of that PR :-)

Reply:
Sounds good to me!